app/app.go

```diff
@@ -126,6 +153,13 @@ func Start(ctx context.Context, cfg config.Config, env env.SpaceEnv) {
 		sync.Stop()
 	}
 
+	if fuseController != nil {
```

Comment:
if fuseController can be nil in some path, it means line 116 can blow up?

Reply:
Ideally, it can't be null, I think I copied something else and just modify for this cleanup 😅. I'll fix it